### PR TITLE
Showing message at the top of upgrade page

### LINKED ISSUE #2369: Showing message at the top of upgrade page
This has been marked as a good candidate for becoming a [first-timers-only issue](http://www.firsttimersonly.com) like [these](https://github.com/publiclab/plots2/labels/first-timers-only), meaning that it's simple, self-contained, and with some extra formatting, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider [reformatting or reposting it as a first-timers-only issue](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors), and then ping @publiclab/reviewers to get it labeled.

****

Pages like this:

https://publiclab.org/notes/stoft/09-16-2016/stability-upgrade-mockup-for-plab-spectrometer-3-0

...are proposed upgrades for a specific project (they have the tag `upgrade:dssk` where "dssk" is the tag name for the project they're upgrading, so we should show a message similar to the one shown on this page:

https://publiclab.org/notes/warren/09-20-2016/building-the-stoft-stability-upgrade-for-desktop-spectrometry-starter-kit-3-0

The notification could be shown on this line, triggering on the power tag `upgrade` instead of `replication`:

https://github.com/publiclab/plots2/blob/3702e83c8f50c86ab6d6a197579990d334a12d90/app/views/notes/show.html.erb#L8

`This is an upgrade for ___________. Try building it and _report back how it goes_.`

"report back..." should link to `#Upgrades` which appears lower on the page. 

[ ] Addition of notice to top of page by adding following condition

`<% if @node.has_power_tag('upgrade') %><div class="alert alert-info"><%= raw t('notes.show.upgrade') %> <a href="/n/<%= @node.power_tag('upgrade') %>"><%= raw t('notes.show.replication_link') %></a>.</div><% end %> `

[ ] Replace `t('notes.show.upgrade')` and `t('notes.show.replication_link')` to match the following text `This is an upgrade for ___________. Try building it and _report back how it goes_.`

[ ] Adding link to upgrades in report back

Feel free to ask for any help...

### PR DESCRIPTION
**Pull request by nagano564**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

fixes #2369 

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
This is looking good -- did the regex change trip the tests? 

Now we need to replace `___________` with the right topic name -- You should be able to use the same `<%= @node.power_tag('upgrade') %>` to fill in that blank!

Thank you!!!!

**Comment by nagano564** [comment]
I am getting this error message with the original regex. I don't know if
it's a problem for others. I also had this same issue when starting the
project.

**Comment by Gauravano** [comment]
## Build Log
The command "tail -n +2 -q ./test/reports/TEST*.xml >> output.xml" exited with 0.
9.04s$ if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then danger --verbose; fi
fatal: Couldn't find remote ref refs/heads/danger_base
fatal: Couldn't find remote ref refs/heads/danger_head
fatal: Couldn't find remote ref refs/heads/danger_base
fatal: Couldn't find remote ref refs/heads/danger_head
fatal: Couldn't find remote ref refs/heads/danger_base
fatal: Couldn't find remote ref refs/heads/danger_head
fatal: Couldn't find remote ref refs/heads/danger_base
fatal: Couldn't find remote ref refs/heads/danger_head
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0
From https://github.com/publiclab/plots2
 * [new tag]           v2.0.1     -> v2.0.1
 * [new tag]           v2.1.0     -> v2.1.0
 * [new tag]           v2.2       -> v2.2
/home/travis/.rvm/gems/ruby-2.4.1/gems/danger-5.5.13/lib/danger/scm_source/git_repo.rb:128:in `find_merge_base': Cannot find a merge base between danger_base and danger_head. (RuntimeError)
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/danger-5.5.13/lib/danger/scm_source/git_repo.rb:16:in `diff_for_folder'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/danger-5.5.13/lib/danger/danger_core/dangerfile.rb:261:in `setup_for_running'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/danger-5.5.13/lib/danger/danger_core/dangerfile.rb:271:in `run'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/danger-5.5.13/lib/danger/danger_core/executor.rb:28:in `run'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/danger-5.5.13/lib/danger/commands/runner.rb:68:in `run'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/danger-5.5.13/bin/danger:5:in `<top (required)>'
	from /home/travis/.rvm/gems/ruby-2.4.1/bin/danger:23:in `load'
	from /home/travis/.rvm/gems/ruby-2.4.1/bin/danger:23:in `<main>'
	from /home/travis/.rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `eval'
	from /home/travis/.rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `<main>'
The command "if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then danger --verbose; fi" exited with 1.

**Comment by Gauravano** [comment]
hi @nagano564, all your tests are passing, the reason build fail is `merge base not found` .

**Comment by Gauravano** [comment]
Could you try opening PR again with new branch ?

**Comment by nagano564** [comment]
I just tried to do it on another PR branch. I will try to read through the
CI errors but I have never done this before so I don't know if I can figure
it out.

**Comment by Gauravano** [comment]
@nagano564 I just visited your plots2 fork and its outdated (80+ commits behind) -- this also causes problem sometimes. You can follow this https://publiclab.org/wiki/contributing-to-public-lab-software#Rewinding+the+master+branch to update your fork's master and then generate PR on new branch.

**Comment by nagano564** [comment]
I don't know what is going on. Can someone pair with me on zoom to figure
this out? I am just going down a huge git rabbit hole

**Comment by Gauravano** [comment]
no worries @nagano564 , tests have passed !!!
The cross is due to codeclimate, which checks for indentation and all, that test has failed but due to previous code, not your code responsible. Thanks for rebasing it.

**Comment by nagano564** [comment]
K.. so will one of the PR be merged??

File: app/views/questions/index.html.erb [hunk 1]
```diff
@@ -13,8 +13,8 @@
     <% cache('feature_questions-header', skip_digest: true) do %>
       <%= feature('questions-header') %>
     <% end %>
-  <% end %>                                   
   </div>
+  <% end %>
```
Comment (Gauravano) [inline app/views/questions/index.html.erb]: nice catch !

File: app/views/notes/show.html.erb [hunk 2]
```diff
@@ -9,6 +9,7 @@
   <% if @node.has_power_tag('series') %><div class="alert alert-info"><%= raw t('notes.show.series') %> <a href="/tag/series:<%= @node.power_tag('series') %>"><%= @node.power_tag('series') %></a>.</div><% end %>
   <% if @node.has_power_tag('build') %><div class="alert alert-info"><%= raw t('notes.show.build') %> <a href="/n/<%= @node.power_tag('build') %>"><%= raw t('notes.show.build_link') %></a>.</div><% end %>
   <% if @node.has_power_tag('with') && current_user && @node.coauthors.collect(&:username).include?(current_user.username) %> <div class="alert alert-success"> You are a coauthor of this post. This attributes your contribution, and gives you edit access to the note. <a href="https://publiclab.org/wiki/power-tags#Coauthorship"> Learn more </a></div><% end %>
+  <% if @node.has_power_tag('upgrade') %><div class="alert alert-success"> This is an upgrade for <%= @node.power_tag('upgrade') %>">. Try building it and <a href="/n/<%= @node.power_tag('upgrade') %>"> report back </a>how it goes.</div><% end %>
```
Comment (Gauravano) [inline app/views/notes/show.html.erb]: Great work :100:

**Comment by Gauravano** [comment]
@jywarren what do you think about code climate issues?

**Comment by Gauravano** [comment]
@nagano564 please close #2724 as issue resolved here. Thanks.

**Comment by nagano564** [comment]
Is there anything else that I need to do to get this PR merged?

**Comment by Gauravano** [comment]
hi @nagano564 , ah you have done your part, it will be merged soon after final review by @jywarren. You can work on other issues if you want, you can find at https://publiclab.github.io/community-toolbox/#r=all . Thanks.

**Comment by jywarren** [comment]
Hi all, thanks for all the hard work. I hope to review today!

**Comment by jywarren** [comment]
Great work!!! :tada:

**Pull request merged**